styles2.css

```diff
@@ -39,49 +38,38 @@ nav:hover {
 division between top bar and the rest
 */
 
-section {
+.port-front {
   background-image: url("images/Ilustration-mobile.png");
```

Comment:
- [ ] The background image it seems that not to be there in the mobile version. Make the image responsive in order to make your design the same as the Figma template.
![Captura de pantalla_2021-09-18_01-30-03](https://user-images.githubusercontent.com/53111464/133878828-c43fcc0d-8819-4bcc-86b0-0e0ae53fb0ea.png)

**Figma**
![Captura de pantalla_2021-09-18_01-30-57](https://user-images.githubusercontent.com/53111464/133878894-ca5f9506-bcd4-4827-bfdb-3411a65a7227.png)

---------------------------------------------------------------------------
![Captura de pantalla_2021-09-18_01-30-29](https://user-images.githubusercontent.com/53111464/133878843-f91d9f67-3010-4b2d-9922-a636db123f95.png)

**Figma**
![Captura de pantalla_2021-09-18_01-31-25](https://user-images.githubusercontent.com/53111464/133878873-249606ca-9414-4a7c-ab1a-7312b44472ec.png)